packages/hardhat-core/src/internal/util/hardforks.ts

```diff
@@ -1,5 +1,34 @@
 import { assertHardhatInvariant } from "../core/errors";
 
+export const BERLIN_EIPS = new Set([
+  // Homestead
+  2, 7, 8,
+  // Tangerine Whistle
+  150, 158,
+  // Spurious Dragon
+  155, 160, 161, 170,
+  // Byzantium
+  100, 140, 196, 197, 198, 211, 214, 649, 658,
+  // Constantinople
+  101, 105, 123, 145,
```

Comment:
Is this correct? According to ethereum.org, it should be:

```suggestion
  1014, 1052, 1234, 145,
```